test/unit/http-get-unit-test.php

```diff
@@ -35,7 +35,16 @@ public function test_that_http_get_throws_exception_with_no_url() {
    */
   public function test_that_http_get_throws_exception_with_bad_website() {
     $http_class = new SearchApi\Commands\HttpGet();
-    $http_class->setUrl( 'BadWebSite' );
-    $http_class->execute();
+    $http_class->setUrl( 'BadWebSite!!^#$%' );
+    var_dump($http_class->execute());
```

Comment:
Why var_dump? I think you want to make an assertion or not output anything directly.